app/code/Magento/Catalog/Model/ProductTypes/GetProductTypeById.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\Catalog\Model\ProductTypes;
+
+use Magento\Catalog\Api\GetProductTypeByIdInterface;
+use Magento\Catalog\Model\ResourceModel\Product as ProductResourceModel;
+
+/**
+ * @inheritdoc
+ */
+class GetProductTypeById implements GetProductTypeByIdInterface
```

Comment:
Implementation should be located in `app/code/Magento/Catalog/Model/ResourceModel/Product`
We don't need to have proxy class without any logic

Reply:
@naydav done.